modules/migration-enabling-pv-resizing-dvm.adoc

```diff
@@ -0,0 +1,64 @@
+// Module included in the following assemblies:
+//
+// * migrating_from_ocp_3_to_4/advanced-migration-options-3-4.adoc
+// * migration-toolkit-for-containers/advanced-migration-options-mtc.adoc
+
+[id="migration-enabling-pv-resizing-dvm_{context}"]
+= Enabling persistent volume resizing for direct volume migration
+
+You can enable persistent volume (PV) resizing for direct volume migration to avoid running out of disk space on the destination cluster.
+
+When the disk usage of a PV reaches a configured level, the `Migration-Controller` custom resource (CR) compares the requested storage capacity of a persistent volume claim (PVC) to its actual provisioned capacity. Then, it calculates the space required on the destination cluster.
+
+A `pv_resizing_threshold` parameter determines when PV resizing is used. The default threshold is `3%`. This means that PV resizing occurs when the disk usage of a PV is more than `97%`. You can increase this threshold so that PV resizing occurs at a lower disk usage level.
```

Comment:
Can you lower the threshold?

Reply:
You can lower the threshold but why would you want to? 97% is considered pretty high.

Reply:
I don't know if there is a use case for it. It just seemed odd to mention that you can increase the threshold and not mention that you can also decrease it. I will leave that up to you.

Reply:
There is no use case for decreasing the threshold. So it does not require documentation.